src/testkit-parser/utils/follow-import.js

```diff
@@ -7,8 +7,10 @@ const followImport = async ({ cwd = '', sourcePath, exportName }) => {
   const finalPath = await resolvePath(cwd, sourcePath);
   const { source } = await readFile(finalPath);
   const ast = parseDriver(source);
-  const exportedNode = await require('./get-exported-node')({ ast, exportName, cwd: path.dirname(finalPath) });
-  return Object.assign(exportedNode, {ast});
+  const scopedCwd = path.dirname(finalPath);
+  const exportedNode = await require('./get-exported-node')({ ast, exportName, cwd: scopedCwd });
```

Comment:
@MSiliunas this is not something that blocks from merging but maybe you investigated how this `require` could be mitigated?